container/src/LuigiContainer.svelte

```diff
@@ -5,6 +5,8 @@
   export let context;
   export let label;
   export let webcomponent;
+  export let can_mfe_initialized; // if `true` at LuigiContainer tag, LuigiContainer sends an event `initialzed` to mfe. Mfe is immediately ready.
+  export let mfe_initialized_ready; // will be set to true if `can_mfe_initialized` is set to `true` otherwise it will be `true` if webcomponent is attatch to the container.
```

Comment:
Could you also add this to `container/public/LuigiContainer.svelte.d.ts` with comments and typings like for the other properties

Reply:
and maybe add the documentation label afterwards so Alex can check it afterwards.